cinder-nanny/scripts/cinder-db-consistency-and-purge.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 SAP SE
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+set -e
+
+unset http_proxy https_proxy all_proxy no_proxy
+
+echo "INFO: copying cinder config files to /etc/cinder"
+cp -vrL /cinder-etc/* /etc/cinder
```

Comment:
Shouldn't this be a deployment issue?  I don't think we should be copying files around in a deployed container.